tensorboard/uploader/uploader_test.py

```diff
@@ -94,7 +98,15 @@ def _create_mock_client():
     return mock_client
 
 
-_SCALARS_ONLY = frozenset((scalars_metadata.PLUGIN_NAME,))
+# By default allow at least one plugin for each upload type: Scalar, Tensor, and
+# Blobs.
+_SCALARS_HISTOGRAMS_AND_GRAPHS = frozenset(
```

Comment:
It just seemed simpler to enable all of scalar, tensor, and blob functionality by default.